packages/cli/src/services/project.service.ee.ts

```diff
@@ -174,25 +176,52 @@ export class ProjectService {
 		return await this.projectRelationRepository.getPersonalProjectOwners(projectIds);
 	}
 
-	async createTeamProject(adminUser: User, data: CreateProjectDto): Promise<Project> {
-		const limit = this.license.getTeamProjectLimit();
-		if (
-			limit !== UNLIMITED_LICENSE_QUOTA &&
-			limit <= (await this.projectRepository.count({ where: { type: 'team' } }))
-		) {
-			throw new TeamProjectOverQuotaError(limit);
+	private async createTeamProjectWithEntityManager(
+		adminUser: User,
+		data: CreateProjectDto,
+		trx: EntityManager,
+	) {
+		const limit = this.licenseState.getMaxTeamProjects();
+
+		const teamProjectCount = await trx.count(Project, { where: { type: 'team' } });
+		if (limit !== UNLIMITED_LICENSE_QUOTA) {
+			if (limit <= teamProjectCount) {
```

Comment:
No need for nested ifs?

Reply:
Good catch, I forgot to move the count back in between during a refactor. 

```ts
		if (limit !== UNLIMITED_LICENSE_QUOTA) {
			const teamProjectCount = await trx.count(Project, { where: { type: 'team' } });
			if (limit <= teamProjectCount) {
```

This way it makes actually sense 😄